docs/dev/app/listings.md

```diff
@@ -1,9 +1,388 @@
+
+**ListingDetails Props**
+- `listingData`: *Listing*. The listing whose ListingDetails will be rendered. The rendering of fields is dependent on the `listing_type` of `listingData`. 
+- (optional) `interpretation`: *boolean*. Default value: false. Should be `true` for Langauge Support listings. Used to distinguish between Case Interpretations and Cases. 
+
+**Example Usage**
+- Langauge Support listings: `<ListingDetails listingData={listingData} interpretation />`
+- Non-Langauge Support listings: `<ListingDetails listingData={listingData} />`
+
+#### How to Customize Fields In ListingDetails 
+**Customizing Fields for Existing Listing Types (i.e. LCA, Language Support or Cases)**
+
+All fields are of type `ListingField<T>[]` (i.e. an array of `ListingField`s), where `ListingField` is defined below: 
+
+```ts
+interface ListingField<T extends Listing> {
+  label: string;
+  getValue: (data: T) => string;
+}
+```
+
+To customize fields for existing listing types, modify the array of fields for the corresponding listing type. 
+- Cases: `caseFields`
+- Case Interpretation: `caseInterpretationFields`
+- Interpretation: `interpretationFields`
+- Document Translation: `docFields`
+- Limited Case Assignments: `lcaFields`
+
+For example, below is how you would add a new field to `lcaFields`. Note that the order of the array determines the order that the fields are rendered. 
+```ts
+const lcaFields: ListingField<LimitedCaseAssignment>[] = [
+    // existing fields ...
+    
+    // New field 
+    {
+        label: 'New Field',
+        getValue: data => data.newfield, 
+        // a function that takes data (type: LimitedCaseAssignments) 
+        // and returns the value of the New Field 
+    },
+];
+```
+
+**Customize Fields for a New Listing Type**
+- Create a a new array of fields of type `ListingField<NewListingType>[]`,where `NewListingType` is a placeholder for the actual listing type.  
+- Add a type check to ensure that `listingFields` returns the fields for your desired listing. 
+```tsx
+const newListingFields: ListingField<NewListingType>[] = [
+    // example field 
+    {
+        label: 'New Field',
+        getValue: data => data.newfield, 
+        // a function that takes data (type: NewListingType) 
+        // and returns the value of the New Field 
+    },
+    // ... other fields 
+];
+
+// ... 
+
+export default function ListingDetails( ... ) {
+    // ... 
+    const listingFields = useMemo(() => {
+        if (listingData.listing_type === 'NewListingType') {
+            return (
+                <ListingFields
+                    fields={newListingFields}
+                    listingData={listingData}
+                />
+            )
+        }
+        // ... handling of other listing types
+    }, [listingData, interpretation]);
+
+    // ... 
+}
+```
+
+#### Changing/Adding Icons 
```

Comment:
this can be helpful outside of listings as well, so we should probably put this in a separate "Miscellaneous" page under "Application"

feel free to start that!

Reply:
sg! added the misc page! 
do we also want to specify when to use webp vs. svg?

Reply:
sure! my intuition is that if the source is a vector graphic, then it should be SVG; otherwise, any regular old image should be converted to webp